src/api/client.ts

```diff
@@ -82,6 +83,16 @@ export function getNostoClient(): PromiseLike<NostoClient> {
     })
 }
 
+export function logAndCaptureError(error: unknown, level: 'debug' | 'info' | 'warn' | 'error') {
+    getNostoClient().then(api => {
+        api.captureError(error, 'nostoAutocomplete', level)
+    })
+    const acceptedLogs = ['debug', 'info', 'warn', 'error']
+    if (level in acceptedLogs) {
```

Comment:
do we need this?

Reply:
I think so. Due to this lib is more for users and I think it would be hard for a client to implement without getting additional logs when something is wrong, so I've added logging to console as well